src/helpers/safeIpc.ts

```diff
@@ -122,5 +125,21 @@ interface SafeIpcRenderer extends Electron.IpcRenderer {
   sendToHost<C extends keyof Channels>(channel: C, ...args: ChannelArgs<C>): void;
 }
 
+interface WebContentsWithSaveIcp extends WebContents {
```

Comment:
Is this supposed to be "safe" instead of "save"?

Reply:
Ahhhh, the typos spellcheck can't detect.